.travis.yml

```diff
@@ -10,6 +10,7 @@ otp_release:
   - 19.0
   - 18.2.1
   - 18.0
+  - 17.4
```

Comment:
Add 20, otherwise the |2 part won't be tested.

Reply:
👍